src/openqaoa-core/problems/tsp.py

```diff
@@ -350,3 +351,111 @@ def qubo(self):
         # Convert to Ising equivalent since variables are in {0, 1} rather than {-1, 1}
         ising_terms, ising_weights = QUBO.convert_qubo_to_ising(n, terms, weights)
         return QUBO(n, ising_terms, ising_weights, self.problem_instance)
+
+class TSP_LP(VRP):
```

Comment:
Is there a reason why `TSP_LP` does not inherit for `TSP` object class instead?

Reply:
Because the linear programming version of the TSP which is the TSP_LP is a specific case of the VRP, or the VRP with just one vehicle. @shahidee44

Reply:
Hmmm I see. Do you think it makes sense for us to rename the class to something like `VRP_LP` instead?
But I guess if the nature of the problem classes are similar this current naming is fine.

Let's include this information in the docstring. i.e. TSP_LP is a specific case of the VRP/its the VRP with just one vehicle.

Reply:
I added a note indicating that the TSP_LP can be seen as a particular case of the VRP for just one vehicle. @shahidee44